### DISCUSSION TITLE
The PowerColor 5750gre 12G Red Devil failed to boot.

### DISCUSSION THREAD

**Discussion by JimmyDewey** [ChefKissInc/NootRX#54] [Q&A] [answered]
I'm sorry to bother you, but as a newcomer, I have a question. I was previously running MacOS on an RX580 graphics card, but recently I switched to a PowerColor 5750gre 12G Red Devil (Device ID: 1002 73DF - 148C 2428). However, when I removed WhatEverGreen and enabled NootRX(Fix RX 6570 -> 6750 #161), the system failed to boot and automatically restarted over and over. The crash report contains the following information. 
> panic(cpu 0 caller 0xffffff8015d413de): NootRX     X6000: @ Failed to apply Navi 22 HwlConvertChipFamily patch
Panicked task 0xffffff86c9780208: 178 threads: pid 0: kernel_task
Backtrace (CPU 0), panicked thread: 0xffffff8b94e560c8, Frame : Return Address
0xffffffec4edff8d0 : 0xffffff8011deb38d mach_kernel : _handle_debugger_trap + 0x4ad
0xffffffec4edff920 : 0xffffff8011f58ed6 mach_kernel : _kdp_i386_trap + 0x116
0xffffffec4edff960 : 0xffffff8011f48120 mach_kernel : _kernel_trap + 0x3e0
0xffffffec4edff9b0 : 0xffffff8011d85951 mach_kernel : _return_from_trap + 0xc1
0xffffffec4edff9d0 : 0xffffff8011deb66d mach_kernel : _DebuggerTrapWithState + 0x5d
0xffffffec4edffac0 : 0xffffff8011dead19 mach_kernel : _panic_trap_to_debugger + 0x1a9
0xffffffec4edffb20 : 0xffffff80125e072b mach_kernel : _panic + 0x84
0xffffffec4edffc10 : 0xffffff8015d413de com.ChefKissInc.NootRX : _NootRX_kern_stop + 0x25e
0xffffffec4edffca0 : 0xffffff8015ce4fdd as.vit9696.Lilu : __ZN7LiluAPI20onEntitlementRequestEPFvPvP4taskPKcRP8OSObjectES0_ + 0x27d
0xffffffec4edffce0 : 0xffffff8015cdce75 as.vit9696.Lilu : __ZN13KernelPatcher18setupKextListeningEv + 0x1835
0xffffffec4edffd30 : 0xffffff8012470379 mach_kernel : __ZN6OSKext4loadEhhP7OSArray + 0x729
0xffffffec4edffd90 : 0xffffff801248683e mach_kernel : __ZN6OSKext22loadKextWithIdentifierEP8OSStringPP8OSObjectbbhhP7OSArray + 0x1ae
0xffffffec4edffde0 : 0xffffff801250ad2e mach_kernel : __ZNK11IOCatalogue14isModuleLoadedEP12OSDictionaryPP8OSObject + 0xee
0xffffffec4edffe30 : 0xffffff80124e7abf mach_kernel : __ZN9IOService15probeCandidatesEP12OSOrderedSet + 0x59f
0xffffffec4edffef0 : 0xffffff80124e7390 mach_kernel : __ZN9IOService14doServiceMatchEj + 0x3b0
0xffffffec4edfff50 : 0xffffff80124ea497 mach_kernel : __ZN15_IOConfigThread4mainEPvi + 0x157
0xffffffec4edfffa0 : 0xffffff8011d8519e mach_kernel : _call_continuation + 0x2e
      Kernel Extensions in backtrace:
         as.vit9696.Lilu(1.6.7)[087C34D2-49F8-3FDA-8757-27E9425C9EE4]@0xffffff8015cda000->0xffffff8015d05fff
         com.ChefKissInc.NootRX(1.0)[EF8A4F76-60EB-3767-9CE4-F8E204EA1C5A]@0xffffff8015d3d000->0xffffff8016347fff
            dependency: as.vit9696.Lilu(1.6.7)[087C34D2-49F8-3FDA-8757-27E9425C9EE4]@0xffffff8015cda000->0xffffff8015d05fff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[802B9CA4-B851-396E-8DA4-B30E751EC48F]@0xffffff8014859000->0xffffff8014888fff
Process name corresponding to current thread (0xffffff8b94e560c8): kernel_task
Boot args: -v keepsyms=1 npci=0x2000 debug=0x100 alcid=1 
Mac OS version:
22D68
Kernel version:
Darwin Kernel Version 22.3.0: Mon Jan 30 20:42:11 PST 2023; root:xnu-8792.81.3~2/RELEASE_X86_64
Kernel UUID: 10E5D254-4A37-3A2A-B560-E6956A093ADE
roots installed: 0
KernelCache slide: 0x0000000011a00000
KernelCache base:  0xffffff8011c00000
Kernel slide:      0x0000000011adc000
Kernel text base:  0xffffff8011cdc000
__HIB  text base: 0xffffff8011b00000
System model name: MacPro7,1 (Mac-27AD2F918AE68F61)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0
System uptime in nanoseconds: 6086021691
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x000000016b16691d
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x0000000d81aa67bc 0x0000000000000000
Compressor Info: 0% of compressed pages limit (OK) and 0% of segments limit (OK) with 0 swapfiles and OK swap space
Zone info:
  Zone map: 0xffffff805fd5c000 - 0xffffffa05fd5c000
  . PGZ   : 0xffffff805fd5c000 - 0xffffff8063d5d000
  . VM    : 0xffffff8063d5d000 - 0xffffff8530090000
  . RO    : 0xffffff8530090000 - 0xffffff86c96f6000
  . GEN0  : 0xffffff86c96f6000 - 0xffffff8b95a29000
  . GEN1  : 0xffffff8b95a29000 - 0xffffff9061d5c000
  . GEN2  : 0xffffff9061d5c000 - 0xffffff952e08f000
  . GEN3  : 0xffffff952e08f000 - 0xffffff99fa3c2000
  . DATA  : 0xffffff99fa3c2000 - 0xffffffa05fd5c000
  Metadata: 0xffffffc8e00e6000 - 0xffffffc9000e6000
  Bitmaps : 0xffffffc9000e6000 - 0xffffffc90c0e6000

Could you please advise me on what steps I should take to ensure that this new graphics card can operate on a Hackintosh system? Thank you very much!

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Your macOS version is too old, do your updates.

**Reply by JimmyDewey** [reply]
Thank you very much! It works!
